xarray/coding/times.py

```diff
@@ -397,15 +407,17 @@ def encode(self, variable, name=None):
 
         return Variable(dims, data, attrs, encoding)
 
-    def decode(self, variable, name=None):
+    def decode(self, variable, name=None, use_cftime=None):
```

Comment:
Should we keep this option on the `__init__` method instead? That’s what we’ve done for other coder objects.

I think that is a little nicer since it makes it more obvious how to use coders.

Reply:
Agreed.  That is nicer.